model/vsc_parser.py

```diff
@@ -305,6 +326,7 @@ def ast_Members(parent, yamltree, argtype = 'members') -> list[Member]:
         node = Member(get_yaml_value(st, 'name'), parent)
         node.description = get_yaml_value(st, 'description')
         node.type = get_yaml_value(st, 'datatype')
```

Comment:
Shall also be changed to datatype?

Reply:
I changed this and squashed it into the original commit and force pushed.